### PR TITLE
Прогрессивное улучшение и базовая оптимизация личного проекта (по желанию)

### PR DESCRIPTION
**Pull request by Ksysha**
--
:boom: https://htmlacademy-htmlcss.github.io/218984-gllacy/5/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -227,82 +225,21 @@ <h3 class="index-content-title">Новое в нашем блоге</h3>
...
-            popupFeedback.classList.remove("popup-feedback-show");
-            overlay.style.display = "none";
-            popupFeedback.classList.remove("popup-feedback-error");
-            popupFeedback.offsetWidth = popupFeedback.offsetWidth;
-          }
-        }
-      });
-
-    </script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```
Comment (zoobestik) [inline index.html]: 😱 Не так не пойдет. Из-за мелкого скрипта, тащить целую библиотеку.

**Comment by zoobestik** [comment]
- [ ]  Нет слова «Хит!» для карточек мороженного
- [ ] Надо добавить [тип шрифта](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L5): [Б10](https://up.htmlacademy.ru/htmlcss/15/criteries#basic-10)
- [ ] слово «любого» [лишнее](https://github.com/Ksysha/218984-gllacy/blob/master/index.html#L127)
- [ ] Текст белый и [не виден](https://github.com/Ksysha/218984-gllacy/blob/master/index.html#L178)
- [ ] Проставь размеры каринкам: [1](https://github.com/Ksysha/218984-gllacy/blob/master/index.html#L86), [2](https://github.com/Ksysha/218984-gllacy/blob/master/index.html#L75), [3](https://github.com/Ksysha/218984-gllacy/blob/master/index.html#L222)
- [ ] `padding: 0` для `.user-list`
- [ ] Обрати внимание: [Д6](https://up.htmlacademy.ru/htmlcss/15/criteries#advanced-6). Для стилизации не использованы. У тебя есть.
- [ ] [Д10](https://up.htmlacademy.ru/htmlcss/15/criteries#advanced-10). Для блока, у которого есть фоновое изображение, прописан фоновый цвет:
[1](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L616), [2](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L619), [3](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L668), [4](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L751), [5](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L768)
- [ ] [Тоже самое](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L651-L654) надо сделать для католога
- [ ] Стиль переопределяется в том же правиле: [1](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L10), [2](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L191), [3](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L1129), [4](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L1174), [5](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L1310)
- [ ] [Тут](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L692) ширина по 48%
- [ ] [Вот тут](https://github.com/Ksysha/218984-gllacy/blob/master/css/style.css#L796) он должен занимать все место минус ширина кнопки. Делается или флексом или `width: calc(100% - 139px);`

**Comment by zoobestik** [comment]
Добавим поправим это и будем готовить к сдаче

**Comment by zoobestik** [comment]
Поищи в css: `box-shadow: inset 0px 0px 0px 2px rgba(255,255,255);` -- битый стиль

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    feedbackForm.classList.add("popup-feedback-error");
+  } else {
+    localStorage.setItem("email", emailFeedback.value);
+    localStorage.setItem("name", nameFeedback.value);
+  }
+});
+
+window.addEventListener("keydown", function(event) {
+  if (event.keyCode === 27) {
+    if (popupFeedback.classList.contains("popup-feedback-show")) {
```
Comment (zoobestik) [inline js/main.js]: `popupFeedback` больше нет, тут ошибка

**Comment by zoobestik** [comment]
Что-то с лейблом для почты. Он виден всегда: 
screen shot 2016-11-22 at 00 30 16

File: css/style.css [hunk 3]
```diff
@@ -1376,8 +1376,8 @@ fieldset {
   width: 478px;
   box-sizing: border-box;
   left: 50%;
-  transform: translateX(-50%);
-  top: 30%;
+  transform: translateX(-50%) translateY(-50%);
+  top: 50%;
```
Comment (zoobestik) [inline css/style.css]: да, так лучше, но теперь анимация надо поправить :/

**Comment by zoobestik** [comment]
**ПОСЛЕ всех правок:**

С **JS** все просто: 
1. Прогоняешь содержимое main.js https://skalman.github.io/UglifyJS-online/
2. Результат сохраняешь рядом в main.min.js

**CSS**:

3. Берешь свой style.css вставляешь в http://csscomb.com/online. Результат вставляешь обратно в style.css
4. Создаешь style.min.css рядом.
4.1 Берешь из style.css и прогоняешь тут через https://autoprefixer.github.io/.
4.2 Результать 3.1 прогоняешь через https://css.github.io/csso/csso.html
4.3 сохраняешь результат в style.min.css

5. Правишь пути в html'ках
